Runtime/Editor/ProjectSettings.cs

```diff
@@ -91,6 +91,26 @@ private void DrawGameSettings()
             if (EditorGUI.EndChangeCheck())
             {
                 gameSettings.domainKey = m_CustomSettings.FindProperty("domainKey").stringValue;
+
+                string domainkey = m_CustomSettings.FindProperty("domainKey").stringValue;
+
+                string pattern = @"(?<=https://)\w+(?=\.api\.lootlocker\.io/)";
+                Regex regex = new Regex(pattern);
+                Match match = regex.Match(domainkey);
+
+                if (match.Success)
+                {
+                    string regexKey = match.Value;
+                    Debug.LogWarning("You accidentally used the domain url instead of the domain key,\nWe took the domain key from the url.: " + regexKey);
```

Comment:
I think we should put this text below the field instead of in the log. See line 116 for an example.

Reply:
I don't agree :P